tests/async/tioselectors.nim

```diff
@@ -142,20 +142,40 @@ when not defined(windows):
     result = true
 
   when ioselSupportedPlatform:
-    proc timer_notification_test(): bool =
+    type TestData = object
+      s1, s2, s3: int
+    proc timer_notification_test_impl(data: var TestData) =
       var selector = newSelector[int]()
-      var timer = selector.registerTimer(100, false, 0)
-      var rc1 = selector.select(140)
-      var rc2 = selector.select(140)
```

Comment:
Just increase this to like 60 secs. `select` should return as soon as the timer is ready anyway, so high times should be fine.

Reply:
this would end up waiting 60s * 20 iterations, ie 20mn before it fails, which is too much.
But I've tweaked the numbers which should be well enough now; if it ever flakes, we can reconsider.
I've also made the one-shot test wait for a different amount for `rc5` since this will be an actual wait time